appium/lib/screen-object.rb

```diff
@@ -146,13 +146,16 @@ def scroll_text_to_view(text, direction = :down, timeout = 40)
   def text_visible?(text, direction)
     if driver.find(text).displayed?
       scroll(direction)
+      sleep 1
```

Comment:
Rather than adding sleep everywhere after scroll, just add sleep to `scroll` method itself at the end.

Also, try it with reduced sleep like - 0.5